62.CAD/Polyline.h

```diff
@@ -216,16 +223,24 @@ class CPolyline
 		{
 			m_sections[m_sections.size() - 1u].count += static_cast<uint32_t>(linePoints.size());
 		}
-		m_linePoints.insert(m_linePoints.end(), linePoints.begin(), linePoints.end());
+
+		constexpr LinePointInfo EMPTY_LINE_POINT_INFO = {};
+		const uint32_t newLinePointSize = m_linePoints.size() + linePoints.size();
+		m_quadBeziers.reserve(newLinePointSize);
+		for (uint32_t i = 0u; i < linePoints.size(); i++)
+		{
```

Comment:
https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/67/files?diff=split&w=0#r1404180657 same applies here